alembic/env.py

```
@@ -0,0 +1,118 @@
+def get_url() -> str:
+    """Get database URL from environment or settings.
+
+    Returns:
+        str: Database connection URL.
+    """
+    # For alembic, we need a sync URL (postgresql:// instead of postgresql+asyncpg://)
+    url = os.environ.get(
+        "DATABASE_URL",
+        "postgresql+asyncpg://fragrance_rater:password@localhost:5432/fragrance_rater",
+    )
+    return url
```

Ruff (0.15.12)

[warning] Unnecessary assignment to `url` before `return` statement

Remove unnecessary assignment

(RET504)